packages/react-charts/src/components/ChartUtils/chart-resize.tsx

```diff
@@ -38,7 +37,7 @@ import { canUseDOM } from './chart-helpers';
  * private observer: any = () => {};
  *
  * public componentDidMount() {
- *   this.observer = getResizeObserver(this.inputRef.current, this.handleResize);
+ *   this.observer = getResizeObserver(this.inputRef, this.handleResize);
```

Comment:
In the examples, we're using `this.containerRef.current`

this.observer = getResizeObserver(this.containerRef.current, this.handleResize);

Reply:
The code editor also uses 
`this.observer = getResizeObserver(this.ref.current, this.handleResize);`

Reply:
oh - oops - i was getting turned around. At one point i tried to keep the getResizeObverver types the same, and these changes would be appropriate, but instead I changed the param types, so I should have reverted these comments.